azure/table_azure_mysql_server.go

```diff
@@ -191,6 +191,13 @@ func tableAzureMySQLServer(_ context.Context) *plugin.Table {
 				Type:        proto.ColumnType_JSON,
 				Transform:   transform.From(extractMySQLServerPrivateEndpointConnections),
 			},
+			{
+				Name:        "server_configurations",
```

Comment:
will it be `Server parameters?` as per this azure [doc](https://docs.microsoft.com/en-us/azure/mysql/howto-configure-audit-logs-portal)